packages/discord.js/typings/index.d.ts

```diff
@@ -2370,8 +2370,12 @@ export class UserSelectMenuInteraction<
     UserSelectMenuComponent | APIUserSelectComponent
   >;
   public componentType: ComponentType.UserSelect;
+  public values: Snowflake[];
   public users: Collection<Snowflake, User>;
-  public members: Collection<Snowflake, CacheTypeReducer<Cached, GuildMember, APIGuildMember>>;
+  public members: Collection<
+    Snowflake,
+    CacheTypeReducer<Cached, GuildMember, APIGuildMember, GuildMember | APIGuildMember, GuildMember | APIGuildMember>
```

Comment:
That's quite a verbose way to say "this is not null".
Maybe these could use `NonNullable<...>` instead?

Reply:
![image](https://user-images.githubusercontent.com/76836550/199573599-f6eee1e4-ca5a-4a25-a324-142e4a7a467e.png)
🤔

Reply:
Seems correct to me, as that will narrow down to `Channel | APIChannel`.

Reply:
but dont look cool

Reply:
w/e it can be like you want to